tests/unit/test_models.py

```diff
@@ -7,20 +9,34 @@
 
 
 class MetadataTest(unittest.TestCase):
+    SPECIAL_CHARS = string.whitespace + string.punctuation.replace("_", "")
+
+    def sanitise_id(self, id_str: str):
+        # An interesting SO answer comparing the performance of several ways of doing this for different inputs:
+        # https://stackoverflow.com/a/27086669
+        for c in string.punctuation.replace("_", ""):
+            if c in id_str:
+                id_str = id_str.replace(c, "")
+        for c in string.whitespace:
+            if c in id_str:
+                id_str = id_str.replace(c, "_")
+        return id_str.lower()
+
     def test_init_defaults(self):
-        test_dataset_name = "test_dataset"
+        test_dataset_name = "MOASA Flight M270"
```

Comment:
Is this the correct name for the flight? I thought they were all MOSSA flights but I could be wrong.

Reply:
I don't know, but I took it from the name of the data files, so if it is wrong, maybe I've misunderstood something?

Reply:
I guess you're probably right then.  It's not even a one-line change if it needs fixing anyway, and we'll find out if it does.